client/app/app.js

```diff
@@ -7,7 +7,10 @@ angular.module('artemis', [
   'ui.bootstrap'
 ])
 .config(function($stateProvider, $urlRouterProvider, $httpProvider) {
-  $urlRouterProvider.otherwise('/menu');
+  $urlRouterProvider.otherwise(function($injector) {
```

Comment:
I was getting an error from the old code where the url would flash repeatedly between some states before angular stopped it and went to `menu` state. This method of defining the `otherwise` fixes that, but doesn't result in anything different. I didn't see this same bug on the deployed version though.